Ghseeli/real week 1

ghseeli opened the pull request:
Here are the updated real analysis notes up to the end of this week. Of need of special review is probably the most recent lecture because I did not understand all of it, especially the proof of the last big theorem. In keeping with the review schedule, I have added @MareoRaft as a reviewer, but @cjl8zf , please feel free to also look at this, especially the specific section mentioned above.

ghseeli committed:
Added real analysis lectures 2 and 3.

ghseeli committed:
Added fourth real analysis lecture.

MareoRaft commented:
so if i want to actually make changes to this, i should accept the pull request and then commit on top of it?

i've never used this feature before.

ghseeli commented:
So, you have a few options. If you want to directly make changes, just check out the branch `ghseeli/real-week-1` and commit on it. Alternatively, you can just make comments on the changes about what needs to be changed and I will do it. Don't merge the pull request, though. Once you have done your review, just give a positive review and then I will look at your changes and if I agree with them, I will merge the PR.

ghseeli commented:
Check this out: https://help.github.com/articles/about-pull-request-reviews/

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -404,5 +404,174 @@ \subsection{Point-Set Topology}
+\subsection{Measurable spaces}
+\begin{defn}
+  Given a set $X$ and a $\sigma$-algebra $\A$ on $X$, we call the pair
+  $(X,\A)$ a \emph{measurable set}.
```
he meant "measurable space"

MareoRaft replied:
An element of A is a "measurable set"

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -404,5 +404,174 @@ \subsection{Point-Set Topology}
+    Linear algebra & Vector spaces & Linear maps \\
+    Group theory & Groups & Group homomorphisms \\
+    Topology & Topological spaces $(X,\Top)$ & Continuous maps \\
+    Measurable sets & $(X,\A)$ & Measurable maps (\S 2.1 Folland) \\
```
again "spaces"

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -404,5 +404,174 @@ \subsection{Point-Set Topology}
+\begin{prop}
+  Let $X = \prod_{\alpha \in A} X_\alpha$ with $(X_\alpha,\M_\alpha)$
+  a family of measurable spaces. Then $\bigotimes_{\alpha \in A} \M_\alpha$ is the
+  smallest $\sigma$-algebra which makes all the natural maps
```
Maybe "projection" maps is a better description than "natural".

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -404,5 +404,174 @@ \subsection{Point-Set Topology}
+  $Y^X = \prod_{\alpha \in A} Y_\alpha$. This means that we can think
+  of a $y \in Y^X$ as $y = (y_\alpha)_{\alpha \in A}$ where $y: A \to
+  \bigcup_{\alpha \in A}$ such that for all $\alpha \in A$, $y(\alpha)
+  = y_\alpha \in Y_\alpha$. 
```
I think it would improve the readability of this paragraph if we just picked $X$ or $A$ and used it throughout, instead of using both.

MareoRaft replied:
Also, can we take the product across A if A is not countable?  I'm wondering how general this product really can be.  I was thinking of (y_alpha) as a sequence, but then realized it may not be.

ghseeli replied:
I agree we should be consistent and use just $X$ after the initial sentence. An uncountable product looks like a function.

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
   set $\P(\{0,1\}^2)$.
 \end{example}
+\subsection*{(1/23/2017) Lecture 2}
+\begin{example}[Product $\sigma$-algebra]
```
This is a definition

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+  \] \[
+    (x_\beta)_{\beta \in A} \mapsto x_\alpha.
+  \]
+  This gives a $\sigma$-algebra on $X$ called the \emph{product
```
Suggest "The *product sigma algebra* on X is the smallest sigma algebra generated by this data.  Namely, ..."

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+    $\sigma$-algebra}, denoted \[
+    \bigotimes_{\alpha \in A} \M_\alpha.
+  \]
+  In more concise terms, we say that \[
```
We can probably just cut out these four lines

ghseeli replied:
I disagree. The previous part is a definition that specifies notation, the latter part is a very condensed mathematical expression. I can rework it to make the first part more related to definition 1.12.

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+\begin{prop}
+  If $A$ is at most countable, then $\bigotimes_{\alpha \in A}
+  \M_\alpha$ is generated by $\prod_{\alpha \in A} E_\alpha$ where
+  $E_\alpha \subset \M_\alpha$. 
```
"\in" instead of "\subset"

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+\end{example}
+\begin{prop}
+  If $A$ is at most countable, then $\bigotimes_{\alpha \in A}
+  \M_\alpha$ is generated by $\prod_{\alpha \in A} E_\alpha$ where
```
It should the the SET of products, where the family of E's can change each time.  That is, it should be \Ep_2 here, not a single element of \Ep_2.

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+      Y = \bigcap_{\alpha \in A} \pi_\alpha^{-1}(E_\alpha).
+    \]
+    So, if $x = (x_\alpha)_{\alpha \in Y}$, then $x \in
+    \pi_\alpha^{-1}(E_\alpha) (\in \Ep_1) \ \iff \ x_\alpha \in
```
suggest reversing order of statements around "iff" for readability

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+  Suppose we have $(x_\alpha)_{\alpha \in A}$ and $\Ep_\alpha \subset
+  \P(X_\alpha)$. Let $\M_\alpha = \M(\Ep_\alpha)$ in $X_\alpha$. Then
+  \begin{itemize}
+  \item $\bigoplus_{\alpha \in A} \M_\alpha = \M(\F_1)$ where $\F_1 =
```
Did you mean the product sigma algebra?  We haven't defined a sum sigma algebra, unless it's being defined here or something...

ghseeli replied:
Haha yes, I did!

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+      \bigoplus_{\alpha \in A} = \M(\F_2)
+    \]
+    with $\F_2 = \{\prod_{\alpha \in A} E_\alpha \mid \forall \alpha
+    \in A, E_\alpha \in \Ep_\alpha\}$
```
nix the "for all alpha".  instead, it should be "for all families (E_alpha) where ever E_alpha in \Ep_\alpha"

ghseeli replied:
Hmmm, this is the exact notation he used in class and I think they are equivalent. I would want to err on the side of what he put on the board when it is not incorrect.

MareoRaft replied:
I don't think "for all alpha" is even sensical.  Haven't we taken the product over all alpha?  So the resulting product is no longer indexed by alpha.  So there is simply no alpha to refer to.

ghseeli replied:
Hmmm. I read that as saying "\F_2 is the set of elements of the form \prod_{\alpha \in A} E_\alpha where we restrict the E_\alpha's such that, for all \alpha \in A, E_\alpha is in \Ep_\alpha." I do agree that perhaps the notation is a little off-putting. Personally, when written in a sentence, I would prefer to say "\F_2 is the set of elements of the form \prod_{\alpha \in A} E_\alpha where we restrict the E_\alpha's such that E_\alpha is in \Ep_\alpha for all \alpha in A," but I guess he chose that ordering because the E_\alpha is dependent on your choice of \alpha, so the choice of \alpha is logically prior?

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+  \]
+\end{cor}
+\begin{proof}[Proof of Corollary]
+  $\R$ is seperable.
```
This made me laugh.

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+\subsection{Point-Set Topology}
+\begin{defn}
+  Let $X$ be a set. A \emph{topology} on $X$ is a set of ``open sets''
+  $\Top \in \P(X)$ such that
```
"subset", not "in"

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+    U \in \Top(d) \ \iff \ \forall x \in U, \exists r > 0 \text{ such
+      that } B(x,r) \subset U.
+  \]
+\end{rmk}
```
This is really a definition of "canonically defined topology" and "equivalent", although the latter is an ambiguous name.

ghseeli replied:
I don't understand the remark about "equivalent." A topology is just a subset of the power set of X. If two topologies are equivalent if they are equivalent as sets.

MareoRaft replied:
Prerequisite:  Given an X, a distance d can be used to generate a topology on X.  We use the distance to define sets of the form B(p, r).  We then use those balls as a basis for the topology.

definition of equivalent:
He's saying that given a fixed X, two distances d1 and d2 are called "equivalent" if they generate exactly the same topology.  

Does that clarify?

ghseeli replied:
Yeah, that clarifies. I think lines 282--283 reading: 
```
If $d_1,d_2$ are two metrics on $X$, we say that $d_1$ and $d_2$ are
equivalent (topologically) if and only if $\Top(d_1) = \Top(d_2)$.
```
Are rigorous enough to clarify that confusion in the notes?

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+  if for all $x \in U$, there exist $k \geq 0$ and $i_1, < \cdots <
+  i_k \in \N$ and $r > 0$ such that \[
+    B = \{y \in X \mid d(x_{i_1},y_{i_1}) < \cdots <
+    d(x_{i_k},y_{i_k}) < r\} \subset U
```
I think it's meant to say that EVERY distance is less than r, not an increasing list of things.

ghseeli replied:
Okay, that makes more sense!

MareoRaft commented on Real Analysis/real-analysis-notes.tex:
```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+\end{prop}
+\begin{proof}
+  For the proof, one notes that a $B$ of this form can also be written \[
+    B = \pi_{i_1}^{-1}(B(x_i,r)) \cap \cdots \cap \pi_{i_k}^{-1}(x_{i_k},r)
```
and so the last thing should be the ball B(x_{i_k},r) or radius r, like the first thing.

MareoRaft committed:
minor changes

MareoRaft reviewed (changes requested):
All of the comments are meant to be addressed.  I also committed some "minor changes" ontop of your latest version.  These are so minor that you don't need to look at them.  So just the 19 comments, even the ones that for some reason say "outdated".

ghseeli commented:
Alright, thanks for the comments! I will commit changes later tonight.

ghseeli committed:
Addressed comments given by @MareoRaft .

ghseeli commented:
Alright, @MareoRaft , I think I have addressed all your comments. Please give it a look when you have time and update your review as you see fit. Once you give a positive review, I will do merge.

MareoRaft reviewed (approved):
I found one last error:  In prop 1.18 (a), on the LHS, it should be the PRODUCT of the borel sets of the Xj.

I haven't looked at your changes specifically, but I'm assuming you followed my suggestions except for the one place where you disagreed.  I trust you so as long as it makes sense to you, i'm happy.

Thanks for your quality work.

ghseeli committed:
Changed the union index to be more explicit.

ghseeli committed:
Addressed all but one of @MareoRaft 's comments.

ghseeli commented:
@MareoRaft , Thank you for all your comments. You definitely did a thorough job catching (important!) mathematical mistakes and, even when we disagree, forced me to think more about and engage with the material. I think this has been a very productive exercise!
